src/ui/src/custom_components/core.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Declare custom components
+ *
+ * @param customComponents list of components in key-value form
+ */
+export function declareCustomComponents(
```

Comment:
I think the function name could be more meaningfull. Maybe `checkCustomComponents`

Reply:
In this case I want to call the function like this `declareCustomComponents`. It seems more logical when you read the calling code :

```typescript
export default declareCustomComponents({
	bubblemessage: BubbleMessage,
	bubbleMessageadvanced: BubbleMessageAdvanced,
});
```